.idea/codeStyles/Project.xml

```diff
@@ -0,0 +1,41 @@
+<component name="ProjectCodeStyleConfiguration">
+  <code_scheme name="Project" version="173">
```

Comment:
Maybe `OpenJFX` is a better name here?

Reply:
I tried to change it but didn't work. It seems IntelliJ expects "Project" there.